bower.json

```diff
@@ -5,10 +5,16 @@
     "dist/rollbar.umd.js"
   ],
   "ignore": [
+    "dist/*.snippet*",
```

Comment:
I just released v1.9.1 which has the snippet file included in the Bower distribution.